iam-proxy-italia-project/conf/backends/pyeudiw_backend.yaml

```diff
@@ -42,80 +42,84 @@ config:
   response_code:
     sym_key: "1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef" # hex string of 64 characters
 
+  security:
+    wallet_attestation_required: true # this is the flag to enable the wallet attestation interoperability, if missing it will be set to true by default
+
   jwt:
     default_sig_alg: ES256 # or RS256. Please note that this signature alg MUST be compliant with the private keys used for the signature. X.509 certificates MUST be therefore ECDSA using ES, and RSA using RS
     default_enc_alg: RSA-OAEP
     default_enc_enc: A256CBC-HS512
     default_exp: 6         # minutes
     enc_alg_supported: &enc_alg_supported
-        - RSA-OAEP
-        - RSA-OAEP-256
-        - ECDH-ES
-        - ECDH-ES+A128KW
-        - ECDH-ES+A192KW
-        - ECDH-ES+A256KW
+      - RSA-OAEP
+      - RSA-OAEP-256
+      - ECDH-ES
+      - ECDH-ES+A128KW
+      - ECDH-ES+A192KW
+      - ECDH-ES+A256KW
     enc_enc_supported: &enc_enc_supported
-        - A128CBC-HS256
-        - A192CBC-HS384
-        - A256CBC-HS512
-        - A128GCM
-        - A192GCM
-        - A256GCM
+      - A128CBC-HS256
+      - A192CBC-HS384
+      - A256CBC-HS512
+      - A128GCM
+      - A192GCM
+      - A256GCM
     sig_alg_supported: &sig_alg_supported
-        - RS256
-        - RS384
-        - RS512
-        - ES256
-        - ES384
-        - ES512
+      - RS256
+      - RS384
+      - RS512
+      - ES256
+      - ES384
+      - ES512
+
 
   authorization:
     client_id: # this field if not set will be autopopulated using internal variables base_url and name using the following format: "<base_url>/<name>"
     auth_iss_id: # this field if not set will be set to client_id in the authz request
     url_scheme: haip
     scopes:
-    - pid-sd-jwt:unique_id+given_name+family_name
+      - pid-sd-jwt:unique_id+given_name+family_name
     default_acr_value: https://www.spid.gov.it/SpidL2
     expiration_time: 5 # minutes
     aud: https://self-issued.me/v2
     response_mode: direct_post
     presentation_definition:
```

Comment:
please remove presentation definition

it is definitively something to abandon, we'll keep it within the sources even if discontinued

please add a note within the readme about it, as discontinued